index.html

```diff
@@ -4,28 +4,31 @@
-        <button id="clearBtn" class="btn btn-gray span-2">C</button>
-        <button id="btn-back" class="btn btn-gray">←</button>
-        <button data-operation="/" id="btn-division" class="btn btn-orange">÷</button>
-        <button data-number="7" id="btn-7" class="btn btn-gray">7</button>
-        <button data-number="8" id="btn-8" class="btn btn-gray">8</button>
-        <button data-number="9" id="btn-9" class="btn btn-gray">9</button>
-        <button data-operation="*" id="btn-multiplication" class="btn btn-orange">×</button>
-        <button data-number="4" id="btn-4" class="btn btn-gray">4</button>
-        <button data-number="5" id="btn-5" class="btn btn-gray">5</button>
-        <button data-number="6" id="btn-6" class="btn btn-gray">6</button>
-        <button data-operation="-" id="btn-subtraction" class="btn btn-orange">-</button>
-        <button data-number="1" id="btn-1" class="btn btn-gray">1</button>
-        <button data-number="2" id="btn-2" class="btn btn-gray">2</button>
-        <button data-number="3" id="btn-3" class="btn btn-gray">3</button>
-        <button data-operation="+" id="btn-addition" class="btn btn-orange">+</button>
-        <button data-number="0" id="btn-0" class="btn btn-gray span-3 ">0</button>
-        <button id="btn-equal" class="btn btn-orange">=</button>
+        <section class="calc-screen text-5xl span-4">0</section>
+        <button id="clearBtn" class="btn text-5xl btn-gray span-2">C</button>
+        <button id="btn-back" class="btn text-5xl btn-gray">←</button>
+        <button data-operation="/" id="btn-division" class="btn text-5xl btn-orange">÷</button>
+        <button data-number="7" id="btn-7" class="btn text-5xl btn-gray">7</button>
+        <button data-number="8" id="btn-8" class="btn text-5xl btn-gray">8</button>
+        <button data-number="9" id="btn-9" class="btn text-5xl btn-gray">9</button>
+        <button data-operation="*" id="btn-multiplication" class="btn text-5xl btn-orange">×</button>
+        <button data-number="4" id="btn-4" class="btn text-5xl btn-gray">4</button>
+        <button data-number="5" id="btn-5" class="btn text-5xl btn-gray">5</button>
+        <button data-number="6" id="btn-6" class="btn text-5xl btn-gray">6</button>
+        <button data-operation="-" id="btn-subtraction" class="btn text-5xl btn-orange">-</button>
+        <button data-number="1" id="btn-1" class="btn text-5xl btn-gray">1</button>
+        <button data-number="2" id="btn-2" class="btn text-5xl btn-gray">2</button>
+        <button data-number="3" id="btn-3" class="btn text-5xl btn-gray">3</button>
+        <button data-operation="+" id="btn-addition" class="btn text-5xl btn-orange">+</button>
+        <button data-number="0" id="btn-0" class="btn text-5xl btn-gray span-3 ">0</button>
+        <button id="btn-equal" class="btn text-5xl btn-orange">=</button>
+        <button class="btn btn-black text-4xl span-4"> Show history </button>
```

Comment:
Let's revert and remove the `utils` file.

Instead let's add a class on the button, and style based on that class.